하주영/정렬/K번째수.js

```diff
@@ -0,0 +1,21 @@
+function solution(array, commands) {
+  let i, j, k;
+  let filtered = []; // [i, j] 범위로 자른 배열
+  var answer = []; // k 번째 수들 배열
+
+  // 자르기
+  commands.forEach((command) => {
+    i = command[0] - 1;
+    j = command[1] - 1;
+    k = command[2] - 1;
```

Comment:
아래처럼 구조분해할당 할 수 있을 것 같아요!

- i, j, k라는 변수를 의미있는 변수명(start, end 등)으로 변경하면 더 가독성 있을 것 같은데 어떠신가요? (코딩테스트는 협업하는 게 아니니 의미를 담지 않아도 된다는 의견도 있지만, 저는 가독성 측면에서 의미가 있으면 더 좋을 것 같아 제안해봅니다!)

```suggestion
  commands.forEach(([i, j, k) => {
```